README.md

```diff
@@ -747,6 +747,44 @@ The default reader `spacy.FewShotReader.v1` supports `.yml`, `.yaml`, `.json` an
 path = "lemma_examples.yml"
 ```
 
+#### spacy.Sentiment.v1
+
+Performs sentiment analysis on provided texts. Scores between 0 and 1 are stored in `Doc._.sentiment` - the higher, the
+more positive. Note in cases of parsing issues (e. g. in case of unexpected LLM responses) the value might be `None0-`
+
+```ini
+[components.llm.task]
+@llm_tasks = "spacy.Sentiment.v2"
+examples = null
+```
+
+| Argument                  | Type                                    | Default                                                  | Description                                                                                                                                           |
+| ------------------------- | --------------------------------------- | -------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `template`                | `str`                                   | [ner.v2.jinja](./spacy_llm/tasks/templates/ner.v2.jinja) | Custom prompt template to send to LLM model. Default templates for each task are located in the `spacy_llm/tasks/templates` directory.                |
```

Comment:
default copy/paste error?

Reply:
Yes, fixed in https://github.com/explosion/spacy-llm/pull/200/commits/02d90478cd8b8ea1354b8eba7fa0d7d8f93331b3.